core/lib/dal/migrations/20250131090112_remove_tx_initiator_address_from_events.up.sql

```diff
@@ -0,0 +1,3 @@
+-- `events.tx_initiator_address` was only used in Block Explorer queries that were removed from the codebase.
+DROP INDEX IF EXISTS events_tx_initiator_address_idx;
```

Comment:
There [was an attempt](https://github.com/matter-labs/zksync-era/blob/ddc953d138456b6cbe2a9bb2f4518bc7800ba0ce/core/lib/dal/migrations/20240306155535_remove_obsolete_indexes.up.sql#L2) to remove this index in an earlier migration, but its name was misspelled unfortunately.